mention-bot.js

```diff
@@ -301,7 +328,8 @@ function guessOwners(
     .filter(function(owner) {
       return config.userBlacklist.indexOf(owner) === -1;
     })
-    .slice(0, 3);
+    .slice(0, config.maxReviewers)
+    .concat(defaultOwners);
```

Comment:
Yep, I can totally see that issue. Quick opinion question, would `mention-bot` want to always notify the `defaultOwners` regardless of `config.maxReviewers`? I would have guessed yes, in which case I would want to filter after adding them.

Thoughts?